app/utils/sendNotifications.js

```diff
@@ -126,17 +128,12 @@ export const getOwners = async (accountAddress, safeAddress) => {
   }
 }
 
-export const getNonce = async (safeAddress) => {
-  const contract = TruffleContract(GnosisSafe)
-  const provider = new Web3.providers.HttpProvider(getNetworkUrl())
-  contract.setProvider(provider)
+export const getNonce = async (tx) => {
+  if (tx.type === 'sendTransaction') {
+    const estimationValue = isTokenTransfer(tx.data) ? '0' : tx.displayedValue.toString(10)
+    const estimations = await getGasEstimation(tx.from, tx.to, estimationValue, tx.data, 0)
 
-  try {
-    const instance = await contract.at(safeAddress)
-    if (!instance) return
-    const nonce = await contract.web3.eth.call({ to: safeAddress, data: '0xaffed0e0' }, 'pending')
-    return contract.web3.toDecimal(nonce).toString()
-  } catch (err) {
-    console.error(err)
+    return estimations && (estimations.nonce + 1).toString()
```

Comment:
I do not know if this message is going to be repeated, but basically, I tried to explain the method: ```getGasEstimation``` returns more than one estimation so, I see two options:
- Change the name of the method
- Create a new name for getting the nonce.